### DISCUSSION TITLE
add label/note to ray entry?

### DISCUSSION THREAD

**Discussion by wivaku** [spatie/ray#453] [General]
Is there a generic way to add a label?

E.g. 

```
ray( now() )->label('current time')
```

Would like to see something like:

> **current time**
> 2021-05-10 09:52:45

That way I can see context of what is being displayed.  
This to avoid confusion in situations like this: `ray($dateOne); ray($dateTwo);`
Don't like to add two entries like this: `ray('current time', now())`.

The functionality provided for a [custom function](https://spatie.be/docs/ray/v1/advanced-usage/adding-your-own-ray-functions-in-php) comes close to that:

```
ray()->sendCustom(now(), 'current time')
```

But that puts it in the space on the left, which is intended for "tags".
And that space on the left has room for short words and a label might be too long for that space.
Btw, would be nice if this is available as `ray( now() )->tag('hey')`

![image](https://user-images.githubusercontent.com/2084872/117625631-8163f100-b176-11eb-8f84-7818ad202f12.png)

**Comment by freekmurze** [comment]
I think it makes sense to add it this, @AdrianMrn and I will take a look at it soon!

**Reply by wivaku** [reply]
that was quick + positive news, thanks Freek and Adrian!
